README.md

```diff
@@ -1,7 +1,7 @@
 
 # Lua-QuickCheck
 
-[![Build Status](https://travis-ci.org/luc-tielen/lua-quickcheck.svg?branch=master)](https://travis-ci.org/luc-tielen/lua-quickcheck)
+[![Build Status](https://github.com/luc-tielen/lua-quickcheck/actions/workflows/test.yml/badge.svg)](https://github.com/luc-tielen/lua-quickcheck/actions/workflows/test.yml)
```

Comment:
I think we are missing a `.github` directory here?

Reply:
Actually no, see an example in another project - https://github.com/ligurio/topology/actions/workflows/test.yml

Reply:
Ok, just making sure 👍